update/views.py

```diff
@@ -13,6 +13,17 @@ def update_metadata(updated_at):
     with open(settings.METADATA_JSON, 'w') as f:
         json.dump(data, f)
 
+def update_tags(updated_at):
+    topics = []
+    data = {'updated_at' : updated_at}
+    for keys in json.load(open(settings.METADATA_JSON)):
+        for topic in keys.split('/'):
+            if topic not in (topics + ['unclassified', 'updated_at']):
```

Comment:
You must ignore `test` and `src` as well.

Reply:
`test` and `src` were not there in the `metadata.json` file and are not included in the `tags.json` file